crates/nu-explore/src/pager/mod.rs

```diff
@@ -793,14 +793,13 @@ fn handle_event<V: View>(
 }
 
 fn handle_exit_key_event(key: &KeyEvent) -> bool {
-    matches!(
-        key,
-        KeyEvent {
-            code: KeyCode::Char('z'),
-            modifiers: KeyModifiers::CONTROL,
-            ..
+    if key.modifiers == KeyModifiers::CONTROL {
+        // these are all common things people might try, might as well handle them all
+        if let KeyCode::Char('c') | KeyCode::Char('d') | KeyCode::Char('q') = key.code {
```

Comment:
Mhh, not so sure if `Ctrl-Q` is as common (I only remember it in the `XOFF`/`XON` pair with `Ctrl-S` for the teletype level shell control)

Reply:
The Micro text editor uses ctrl + q; I think because it’s similar to the command+q shortcut on macOS.

Reply:
I kinda like `ctrl+q` but it is less traditional than the others, can remove it if you like.

Reply:
afaik ctrl + q is quite common to quit an application, browsers use it often, Zellij does too, ...

Reply:
Ok then let's go ahead with it